spec/hangman_spec.rb

```diff
@@ -0,0 +1,61 @@
+# frozen_string_literal: true
+
+require './hangman'
+
+RSpec.describe Hangman do
+  describe ".initialize" do
+    let!(:subject) { described_class.new }
```

Comment:
`Discussion`
You shouldn't need to use `let!` here (or much at all actually).

I've personally in 3 years of writing RSpec tests written `let!` maybe 5 times, 3 of which were eventually removed.
Let's see why this is happening and whether or not:
 - we can re-write it without loading it first, OR
 - it's pointing out something about your code design